meetings/2025-05-13-taskforce-meeting.md

```diff
@@ -0,0 +1,87 @@
+- Darius: what is, data-wise, the difference between content warnings and labels/annotations?
+- Emelia: Content warnings are free text. For labels, there would be something like a label vocabulary that is standardized.
+- Darius: so content warnings would be text in a field; labels would be Objects that can be referenced and embedded in other contexts.
+- a: so would "label" = "structured linked data reference", while "warning" = "unstructured freeform text"? 
+- Jaz: this sounds like we are formalizing the user experience we've observed for content warnings on mastodon, which I find unfit for purpose. The fact that one platform does something with collapsing... a well structured model for this stuff would allow for new emergent activites based on the data. I tentatively agree that a preference would be for vocabularies to exist that clients could make use of to help labeling and warning, but I am itching at the thought that we are formalizing an observed experience from one microblogging software. Instead of trying to put formality around the thing we see and the way it's seen in certain clients, if we unravel and rewind that and think about it from an agnostic perspective we can come up with a way to provide developers ways to come up with what they need while giving audiences more freedom too. 
+- Darius: I largely agree with Jaz. How does this work in ATProto with content labelers?
+- Jaz: You bring your own vocabulary, and the PDS can apply some number of actions... hide, blur, warn, maybe some other things. Labeler A can say "depiction of war" and "blur" and Labeler B can say "war images" and "hide". 
+- Emelia: https://docs.joinmastodon.org/entities/Filter/#filter_action mastodon now has the same three actions with content filtering 
+- Emelia: the difference between a label and content warning is the former is structured linked data that can be referenced, latter is unstrutured freeform text. For labels you have structured labels, but I'm not saying there should be one vocabulary but many vocabularies. Bluesky has a base set of labels that they have defined, Mastodon would probably also want to have theirs.
+- Darius: so a vocabulary should be an Unordered Collection of Label objects.
+- Emelia: yes, you can see it in FIRES: https://fires.fedimod.org/reference/protocol/labels.html
+- Emelia: I don't want us to be defining the labels. Just how you define the labels and share them, and how you associate a label with an object in the AP spec. You could cram all this into web annotations but I don't think that makes sense because we would be extending the web annotations vocabulary anyway since it doesn't have a notion of labels.
```

Comment:
```suggestion
- Emelia: yes, you can [see it in FIRES](https://fires.fedimod.org/reference/protocol/labels.html).
- ... I don't want us to be defining the _labels_. Just _how you define and share_ the labels, and _how you associate_ a label with an object in the AP spec. You could cram all this into Web Annotations, but I don't think that makes sense because we would be extending the Web Annotations vocabulary anyway, since it doesn't have a notion of labels.
```